.github/ISSUE_TEMPLATE/zFeatureRequest.md

```diff
@@ -4,7 +4,36 @@ about: Is there additional functionality you would love us to consider?
 ---
 
 <!--
-Please observe [CONTRIBUTING.md](https://github.com/chocolatey/.github/blob/main/CONTRIBUTING.md) guidelines prior to creating your issue.
+Ensure you have read over [Submitting Issues](https://github.com/chocolatey/.github/blob/main/SUBMITTING_ISSUES.md)
+
+Please check to see if your issue already exists with a quick search of the issues. Start with one relevant term and then add if you get too many results.
 
 NOTE: Keep in mind we have a [Code Of Conduct](https://github.com/chocolatey/.github/blob/main/CODE_OF_CONDUCT.md) that we expect folks to observe when they are looking for support in the Chocolatey community.
+
+Name your issue appropriately: give it a sentence that reads well enough for anyone seeing this in the release notes to what it is.
+
+When writing out the issue details please ensure you are writing it as if you were explaining it to somebody else.
+Even if you will be working on and resolving the issue yourself. This helps others to understand the reasons for the issue and for it to be searchable in future.
+
+Please do not remove any of the headings.
+If a heading is not applicable then enter N/A: Why it's not applicable
+
+Please remove all comments before submitting.
 -->
+
+## Is Your Feature Request Related To A Problem? Please describe.
+
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
+
+## Describe The Solution. Why is it needed?
+
+<!-- A clear and concise description of what you want to happen. -->
+
+## Additional Context.
+
+<!-- Add any other context or screenshots about the feature request here. -->
+
+## Related Issues
+
+<!-- Enter and link any related issues here. -->
+#
```

Comment:
This item would translate to a `h1` header if someone leaves it in without either removing it or adding an issue number.

Should we clarify in the comment above to replace it with `N/A` if there are no related issues (someone will definitely miss the mention on line 19)?

Reply:
Fixed in latest commit!